pkg/performanceprofile/controller/performanceprofile/status/status.go

```diff
@@ -32,6 +32,13 @@ const (
 	ConditionFailedGettingTunedProfileStatus = "GettingTunedStatusFailed"
 )
 
+type Writer interface {
+	// Update updates the status reported by the controller
+	Update(ctx context.Context, object client.Object, conditions []conditionsv1.Condition) error
+	// UpdateOwnedConditions updates the conditions of the components owned by the controller
+	UpdateOwnedConditions(ctx context.Context, object client.Object, profileMCP *mcov1.MachineConfigPool) error
```

Comment:
In the implementation for `UpdateOwnedConditions` in hypershift, are we counting on calling this function with an empty  `*mcov1.MachineConfigPool` (we said no MCP on hypershift IIRC)?

Reply:
this is a good point indeed

Reply:
Another option is to get the MCP from within the function

Reply:
Done